email/resource_email.go

```diff
@@ -64,6 +65,17 @@ func resourceEmail() *schema.Resource {
 	}
 }
 
+// regex function to extract the status code
+func extractStatusCode(errMsg string) string {
+	// Regex to find the first three-digit number, which is the SMTP status code
+	re := regexp.MustCompile(`\b\d{3}\b`)
```

Comment:
Can we check for the `Error: ` prefix here? This is so that we don't take any random 3 digit number from the message.

Reply:
the errMsg begins with the error code `Error: ` is output formatting, so we should be okay?